wgengine/winnet/winnet.go

```diff
@@ -45,6 +46,23 @@ type INetwork struct {
 	ole.IDispatch
 }
 
+type INetworkVtbl struct {
+	ole.IDispatchVtbl
+	GetName                    uintptr
+	SetName                    uintptr
+	GetDescription             uintptr
+	SetDescription             uintptr
+	GetNetworkId               uintptr
+	GetDomainType              uintptr
+	GetNetworkConnections      uintptr
+	GetTimeCreatedAndConnected uintptr
+	Get_IsConnectedToInternet  uintptr
+	Get_IsConnected            uintptr
+	GetConnectivity            uintptr
+	GetCategory                uintptr
+	SetCategory                uintptr
+}
```

Comment:
This appears to match the vtable that Microsoft generates for their Rust API bindings (my go-to for finding the structure for things without needing to dig through C++):
https://github.com/microsoft/windows-rs/blob/962d0a9f19d049a37f704b8ea3c7ab7efd1d95cb/crates/libs/windows/src/Windows/Win32/Networking/NetworkListManager/mod.rs#L248-L269